drivers/virtualbox/virtualbox_windows.go

```diff
@@ -95,6 +95,22 @@ func getShareDriveAndName() (string, string) {
 }
 
 func isHyperVInstalled() bool {
+	// check if hyper-v is installed
 	_, err := exec.LookPath("vmms.exe")
-	return err == nil
+	if err == nil{
+		// check to see if a hypervisor is present. if hyper-v is installed and enabled,
+		// display an error explaining the incompatibility between virtualbox and hyper-v.
+		output, err := cmdOutput("wmic", "computersystem", "get", "hypervisorpresent")
+
+		if err != nil {
+			errmsg := "Could not check to see if Hyper-V is running."
+			log.Debugf(errmsg, err)
+			return false
+		}
+
+		enabled := strings.Contains(output, "TRUE")
+		return enabled
+	} else{
```

Comment:
@theodthompson  You don't need the `else` here. And it's better to deal with errors first with a guard.